client/tests/integration/offline_peers.rs

```diff
@@ -7,11 +7,11 @@ use tokio::runtime::Runtime;
 
 #[test]
 fn genesis_block_is_commited_with_some_offline_peers() {
-    prepare_test_for_nextest!();
     // Given
     let rt = Runtime::test();
 
-    let (network, iroha_client) = rt.block_on(<Network>::start_test_with_offline(4, 1, 1));
+    let (network, iroha_client) =
+        rt.block_on(<Network>::start_test_with_offline_and_port_and_set_n_shifts(4, 1, 1, 10_560));
```

Comment:
This could become 

```suggestion
test_network! {
	offline_peers: 1, 
	total_peers: 4, 
	n_shifts: 1, 
	port: 10_560
}
```
and give us `network` and `iroha_client` in scope as a side-effect.

Reply:
is it possible due to hygiene? ([playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=565508d01125e46c65c04bb1614be50b))

Reply:
https://gist.github.com/Kestrer/8c05ebd4e0e9347eb05f265dfb7252e1#procedural-macros-use-mixed_site-where-possible

Reply:
Sure, i was confused by declarative macro.

Reply:
Not going this route as we've discussed that the solution is good as it is, not worth a separate new crate with a non-hygenic macro for now.